### PR TITLE
feat: adding VSCode settings folder(`.vscode/`) to `.gitignore`

### PR DESCRIPTION
**Pull request by kiper220-alt**

### CODE REVIEW

File: .clang_format
Comment (august-alt) [inline .clang_format:1]: While there is `.clang_format` in addition to `.clang-format`?

File: .clang-format
Comment (august-alt) [inline .clang-format:1]: We should move this file to a separate PR.

File: .clang_format
Comment (kiper220-alt) [reply .clang_format:1]: git did not pick up the file deletion. Maybe I missed it.

**Review by august-alt — APPROVED** [review]
